test/powershell/Language/Operators/ReplaceOperator.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+Describe "Replace Operator" -Tags CI {
+    Context "Replace operator" {
+        It "Replace operator can replace string values using regular expressions" {
+            $res = "Get-Process" -replace "Get", "Stop"
+            $res | Should Be "Stop-Process"
+
+            $res = "image.gif" -replace "\.gif$",".jpg"
```

Comment:
Why we need the test? Looks as dup line 4. Do it test another code path?

Reply:
First test is the first and simplest example from the `about_Comparison_Operators` help file, this one is the first example that shows it's actually a regex operator and not just static string replacement.